### PR TITLE
Implement Capella Fork

### PR DESCRIPTION
**Pull request by marioevz**
Adds the default fork version and epoch for capella fork, from [ethereum/consensus-specs
](https://github.com/ethereum/consensus-specs/), as well as other placeholders for the fork information.

Currently required on hive tests to be able to start eth2/engine simulator with Nimbus beacon node.

### CODE REVIEW

File: eth2/beacon/common/spec.go [hunk 1]
```diff
@@ -108,6 +108,9 @@ type BellatrixPreset struct {
 	MAX_EXTRA_DATA_BYTES                       uint64 `yaml:"MAX_EXTRA_DATA_BYTES" json:"MAX_EXTRA_DATA_BYTES"`
 }
 
+type CapellaPreset struct {
```
Comment (protolambda) [inline eth2/beacon/common/spec.go:111]: We should add these presets: https://github.com/ethereum/consensus-specs/blob/dev/presets/mainnet/capella.yaml

Comment (marioevz) [reply eth2/beacon/common/spec.go:111]: Thanks! I was looking at the empty files of the master branch and missed these files.

**Comment by marioevz** [comment]
Latest commit fully implements Capella, but some of the tests are still failing, I need to debug further.
@protolambda

**Comment by protolambda** [comment]
Implemented a bunch of fixes, but the ssz serialization of BeaconBlockBody (and all types it is contained in) is still failing, so I missed something there. Will continue later today.

**Comment by protolambda** [comment]
Made more fixes + added test runners and updated test fork settings. All tests are passing now, including new Capella tests. Thank you for contributing a lot of the Capella functionality here :+1:

**Pull request merged**